webextensions/common/constants.js

```diff
@@ -319,3 +318,5 @@ export const kSHORTHAND_URIS = {
 };
 
 export const kNOTIFICATION_DEFAULT_ICON = '/resources/icon64.png';
+
+export const kTAB_CLOSE_BOX_ELEMENT = 'tst-tab-closebox';
```

Comment:
The prefix `tst-` looks unnecessary for me, because the namespace and the document namespace is already isolated for TST.

Reply:
@piroor okay. I removed it in https://github.com/piroor/treestyletab/pull/2392/commits/16beebe03857d0162b83e8a42f0ab04f232f73f9

Reply:
Oops, with the same reason `TST` prefix for classes and file names also kook unnecessary for me. (Did you give the prefix for a future usecase reusing the file in other addons?)

Reply:
> Oops, with the same reason TST prefix for classes and file names also look unnecessary for me. (Did you give the prefix for a future usecase reusing the file in other addons?)

Sorry. I forgot to rename them. I'll change in the next patch.

Reply:
I've renamed them https://github.com/piroor/treestyletab/pull/2392/commits/fae20fbb29ef1f7a62da47550547f4b22b9f64cc